pandas/core/frame.py

```diff
@@ -5980,6 +5980,49 @@ def melt(self, id_vars=None, value_vars=None, var_name=None,
                     var_name=var_name, value_name=value_name,
                     col_level=col_level)
 
+    def explode(self, col_name, sep=None, dtype=None):
+        """
+        Create a new DataFrame where each element in each row
+        of a list-like column `col_name` is expanded to its own row
+
+        .. versionadded:: 0.24.0
+
+        Parameters
+        ----------
+        col_name : str
+            Name of the column to be exploded
+        sep : str, default None
+            Convenience to split a string `col_name` before exploding
+        dtype : str or dtype, default None
+            Optionally coerce the dtype of exploded column
+-
```

Comment:
Stray character here?

Reply:
LOL, at first glance I thought that was a git diff acting weird. I clearly forgot to validate the docstring